test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1542,16 +1557,15 @@ UCS_TEST_P(test_ucp_sockaddr_protocols, am_zcopy_64k,
 }
 
 
-
 /* For DC case, allow fallback to UD if DC is not supported */
 #define UCP_INSTANTIATE_CM_TEST_CASE(_test_case) \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, dcudx, "dc_x,ud") \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, ud,    "ud_v") \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, udx,   "ud_x") \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, rc,    "rc_v") \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, rcx,   "rc_x") \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, ib,    "ib")   \
-    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, tcp,   "tcp")  \
+    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, dcudx, "dc_x,ud," UCP_TEST_GPU_COPY_TLS) \
```

Comment:
can we reuse UCP_INSTANTIATE_TEST_CASE_GPU_AWARE?

Reply:
unfortunately, they have different sets of TLs instantiated

Reply:
is it possible to unite some common parts?

Reply:
done

Reply:
@yosefe is it ok to have `UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE` macro to instantiate a test case with GPU support?
I didn't find a better way to reuse `UCP_INSTANTIATE_TEST_CASE_GPU_AWARE` macro for CM instantiation macro.